app/controllers/users/dossiers_controller.rb

```diff
@@ -373,6 +373,10 @@ def change_groupe_instructeur?
       end
     end
 
+    def missing_groupe_instructeur?
```

Comment:
J'essaye un autre wording mais c'est pas top non plus.

```suggestion
    def should_reset_group_instructeur?
```

Reply:
J'ai fait ce nom par symétrie avec `change_groupe_instructeur?` dans le même controller. Aussi je pense que c'est pas un "reset".
Je peux proposer `should_change_groupe_instructeur?` et `should_fill_groupe_instructeur?` si tu préfères.